src/marqo/tensor_search/constants.py

```diff
@@ -27,3 +27,8 @@
 NON_OFFICIAL_LUCENE_SPECIAL_CHARS = {
     ' '
 }
+
+# Order is important here: transient, persistent, then defaults.
+OPENSEARCH_CLUSTER_SETTINGS_TYPES = ["transient", "persistent", "defaults"]
+
+BYTE_SUFFIXES = ['b', 'kb', 'mb', 'gb', 'tb', 'pb', 'eb', 'zb', 'yb']
```

Comment:
This is really elegant, but at the same time the use of index as exponent worries me a bit, since if we later changed this list for any reason, the bug introduced would be really hard to catch (we'd get no error, just unexpected behaviour). I suggest replacing this with a `dict` even though the code won't look as nice as your approach.

Reply:
done, implemented your suggestion!